gpt-survey-summarizer/values.yaml

```diff
@@ -21,14 +24,39 @@ redis:
   auth:
     # -- Redis password
     password: ""
+  # -- Configuration to add to redis.conf
+  commonConfiguration: |-
```

Comment:
**chore**:
This change will force us to redeploy the redis istance.

We must not forget to manually dump the database and provision it after redeployment.

Reply:
Don't we have persistence enabled on the redis volume?

Reply:
We do. Thought it would be nice to continue the DB from where it left off.

Reply:
I agree. But that should be the case by default. When reconfiguring the `redis` deployment, a new pod should be scheduled using the same PVC/PV. So the data should be available in the `redis` after this PR is merged?